pkg/crc/segment/segment.go

```diff
@@ -0,0 +1,74 @@
+package segment
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/version"
+	"github.com/pborman/uuid"
+	"github.com/segmentio/analytics-go"
+)
+
+func NewSegmentClient() (analytics.Client, error) {
+	return analytics.NewWithConfig("cvpHsNcmGCJqVzf6YxrSnVlwFSAZaYtp", analytics.Config{
+		DefaultContext: &analytics.Context{
+			App: analytics.AppInfo{
+				Name:    constants.DefaultName,
+				Version: version.GetCRCVersion(),
+			},
+		},
+	})
+}
+
+func UploadToSegment(client analytics.Client, err error) error {
```

Comment:
`ToSegment` is also redundant imo. Could be `ReportError` though, as this is not uploading arbitrary data, just reporting an error to segment.

Reply:
This contain logs and error.